contrib/metadata-types/src/types/HasuraMetadataV3.ts

```diff
@@ -9,6 +9,8 @@ import {
   TableEntry,
 } from "./HasuraMetadataV2"
 
+export * from "./HasuraMetadataV2"
```

Comment:
This seems like the best, non-breaking option:

1. target this `HasuraMetadataV3` module as the main entry
2. re-export all `HasuraMetadataV2` things